src/bundle/Resources/public/scss/_header-user-menu.scss

```diff
@@ -8,6 +8,7 @@
     &__popup-menu {
         z-index: 1060;
         top: calculateRem(53px);
+        right: calculateRem(20px);
```

Comment:
Note: when clicking the toggle the popup was changing position/width causing misclick. Setting `right` fixes this problem.